commands/project/dev/index.ts

```diff
@@ -115,18 +122,31 @@ function projectDevBuilder(yargs: Argv): Argv<ProjectDevArgs> {
   yargs.option('profile', {
     type: 'string',
     alias: 'p',
-    description: i18n(`commands.project.subcommands.dev.options.profile`),
+    description: commands.project.dev.options.profile,
+    hidden: true,
+  });
+
+  yargs.options('targetTestingAccount', {
+    type: 'string',
+    // TODO: We may want to add messaging that this only works on certain platform versions
+    description: commands.project.dev.options.targetTestingAccount,
+    hidden: true,
+  });
+
+  yargs.options('targetProjectAccount', {
```

Comment:
Nit: The naming of these is very similar.  I wonder if we should differentiate them more.  Maybe just removing the target prefix.  Reviewing this PR I found myself doing double takes a lot.